lib/src/services/movie-service.dart

```diff
@@ -69,9 +91,12 @@ abstract class MovieService {
   Future<Response> getWithResilience(Uri uri);
 
   Future<PagedResult<MovieBase>> _fetchPagedResult(
-      MovieSearchSettings settings, String url) async {
+    MovieSearchSettings settings,
+    String url,
+  ) async {
     var queryParams = settings.toJson()..["api_key"] = _apiKey;
     Uri uri = _buildUri(url, queryParams);
+    print(uri.toString());
```

Comment:
this is probably redundant

Reply:
Ahh sorry. I was using it while debugging and forgot to remove later. 

I'll delete it.